baseapp/baseapp.go

```diff
@@ -223,12 +223,22 @@ func (app *BaseApp) Name() string {
 	return app.name
 }
 
-func (app *BaseApp) InitAppVersion(ctx sdk.Context) {
```

Comment:
[question] is the removal of this public method _technically_ breaking? I don't think we need to cut a major release of Cosmos SDK as long as know that celestia-app has to replace usage of `InitAppVersion` when it bumps to a new release of Cosmos SDK with this PR.

Side note: Is this change back-portable to v1.x? If it isn't, then we may need to create a new branch in this repo if there's a bug we need to address on celestia-app v1.x after this PR merges. Perhaps not worth worrying about for now.

Reply:
I could keep this function around if we were worried about breaking changes. I had intended to replace the usage of `InitAppVersion`.

Reply:
I'm more concerned with the ability to backport than the breaking change. IMO if it's possible to backport to celestia-app v1.x easily then I'm fine with removing `InitAppVersion`. If it isn't possible to backport then we may have to maintain multiple Cosmos SDK branches (one per celestia-app major version) which seems like it could be a maintenance burden.

Reply:
It's not used in v1.x so it should be easy to backport